pdns/recursordist/docs/lua-config/ztc.rst

```diff
@@ -15,6 +15,26 @@ To load the root zone from Internic into the recursor once at startup and when t
 
      zoneToCache(".", "url", "https://www.internic.net/domain/root.zone", { refreshPeriod = 0 })
 
+DNSSEC and ZONEMD validation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Starting with version 4.7.0, the Recursor will do validation of the zone retrieved.
+Validation consists of two parts: ``DNSSEC`` and ``ZONEMD``.
+``ZONEMD`` is described in :rfc:`8976`.
+
+For the ``DNSSEC`` part, if the global :ref:`setting-dnssec` setting is not ``off`` or ``process-no-validate`` and the `DS` record from the parent zone or trust anchor indicates the zone is ``DNSSEC`` signed, the recursor will validate the ``DNSKEY`` records of the zone.
+If a ``ZONEMD`` record is present, it will also validate the ``ZONEMD`` record.
+If no ``ZONEMD`` is present, the ``NSEC`` or ``NSEC3`` denial of the ``ZONEMD`` record will be validated.
+Note that this is not a full validation of all signatures.
+The signatures of the remaining records will be verified on-demand once the records are inserted into the cache.
```

Comment:
"inserted into", or "retrieved from and found to be in Indeterminate state"?

Reply:
A bit unclear indeed, I'll reformulate.